pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -2977,6 +2977,8 @@ type StorageClassDeviceSet struct {
 
 // CephFilesystemSubVolumeGroup represents a Ceph Filesystem SubVolumeGroup
 // +kubebuilder:printcolumn:name="Phase",type=string,JSONPath=`.status.phase`
+// +kubebuilder:printcolumn:name="Filesystem",type=string,JSONPath=`.spec.filesystemName`,description="Name of the CephFileSystem"
+// +kubebuilder:printcolumn:name="Quota",type=string,JSONPath=`.spec.quota`
```

Comment:
please add the pinning status too with priority=1

Reply:
@parth-gr Well I am confused now :smile: 
I would like to do that, but in the comment above you said that it should not be in the status. 
So maybe a stupid question but what way do you think it would be best to add it here, if not through the status method I had implemented earlier?

Reply:
we can read it from the spec right `.spec.pinning`?

Reply:
Maybe try to use the type:`string` and format `byte` https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.0.md#data-types
For printing object, `.spec.pinning`, otherwise fine to add the `status method` back. 
@Madhu-1 any other suggestion?

Reply:
Yes exactly or at least to my knowledge these would then be the two options when reading directly from pinning spec

```
$ kubectl get cephfilesystemsubvolumegroups.ceph.rook.io -A -o wide
NAMESPACE   NAME      PHASE         FILESYSTEM   QUOTA   PINNING             AGE
rook-ceph   group-a   Progressing   myfs                 {"distributed":1}   11s

```

or
```
$ kubectl get cephfilesystemsubvolumegroups.ceph.rook.io -A -o wide
NAMESPACE   NAME      PHASE         FILESYSTEM   QUOTA   DISTRIBUTED    RANDOM    EXPORT    AGE
rook-ceph   group-a   Progressing   myfs                 1                                  11s

```

Reply:
The first option looks good, maybe we can format it better....

Reply:
Seems like I was a bit to eager to celebrate as the custom format functions did not work as expected and all roads pointed me back to custom status field for custom formatting of a field @parth-gr

So the options I currently see is we live with {"distributed": 1} or we can take it through the status.info field for custom formatting: distributed=1

Reply:
The point of printing this info in the additional columns is to make it easily human readable. Since JSON is not readable for most humans, we need a custom formatted field that we generate under the status. So that would be a column like this?
```
PINNING
distributed=1
```

Reply:
So for that we need to update the CRD status too with pinning details, would that be fine?

As Kubebuilder itself doesn't support custom formatting

Reply:
Yes, we can add the CRD status. It's been done in other places similarly such as #13958.